### ISSUE TITLE
[Feature request] Ability to rewind for a peirod of time

### ISSUE THREAD

**Issue by Explorare** [glubsy/livestream_saver#32]
Is it possible to specify the rewind time and recording duration? It would helps a lot while making real time short clips.

**Comment by glubsy** [comment]
Yes, I think this should be possible. Might be something worth looking into.
